src/meltano/cloud/cli/config.py

```diff
@@ -119,11 +240,502 @@ async def set_value(
+
+        Args:
+            context: The click context
+            recipient: The recipient of notification to be updated
+            new: The new recipient value
+        """
+        if not new and not status:
+            click.echo(
+                "Nothing to update. Please provide either --new or --status "
+                "option to update the notification\n",
+            )
+            return
+
+        update_recipient = prompt_and_validate(
+            value=recipient,
+            type=type,
+            prompt_message=(
+                f"Please provide old {type} recipient for " "notification to be updated"
+            ),
+            error_message=f"Invalid {type} value",
+        )
+
+        new_key = None
+        if new:
+            new_key = prompt_and_validate(
+                value=new,
+                type=type,
+                prompt_message=f"Please provide new {type} value to update to",
+                error_message=f"Invalid {type} value",
+            )
+
+        async with ConfigCloudClient(config=context.config) as client:
+            try:
+                await client.update_notification(
+                    recipient=update_recipient,
+                    new_key=new_key,
+                    status=status,
+                )
+                click.echo(f"Successfully updated {type} notification")
+            except Exception as e:
+                raise click.ClickException(
+                    f"{e}. \n\n"
+                    "Please ensure you've correctly followed instructions at: "
+                    "https://docs.meltano.com/cloud/cloud-cli\n"
+                    "If the issue persists, "
+                    "we'd be happy to help at: https://meltano.com/slack",
+                ) from None
+
+    update_notification.__name__ = f"update_notification_{type}"
+    return update_notification
+
+
+create_update_command("webhook")
+create_update_command("email")
+
+
+@notification.group()
+def delete() -> None:
+    """Delete a meltano cloud notification."""
+
+
+def create_delete_command(type: t.Literal["webhook", "email"]):
+    """Helper function to create delete commands for specific type.
+
+    Args:
+        type: The type of notification
+    """
+
+    @delete.command(type, help=f"Delete {type} notification")
+    @click.option(
+        "--recipient",
+        nargs=1,
+        type=str,
+        help=f"{type} recipient to delete notification for",
+    )
+    @pass_context
+    @run_async
+    async def delete_notification(
+        context: MeltanoCloudCLIContext,
+        recipient: str | None,
+    ):
+        """Delete specific notification type.
+
+        Args:
+            context: The click context
+            recipient: The recipient for notification
+        """
+        input_value = prompt_and_validate(
+            value=recipient,
+            type=type,
+            prompt_message=f"Please provide a {type} to delete",
```

Comment:
It's not clear what should be specified when given this prompt. The message should make it clear that we're looking for one of the items in the "recipient" column of the `notification list` command. I'm not sure how to concisely phrase that.